analyses/oncoprint-landscape/tables/primary-plus_sample_n_in_oncoprint.tsv

```diff
@@ -0,0 +1,5 @@
+broad_histology	n
```

Comment:
Noting here that these numbers are the same as in `analyses/oncoprint-landscape/tables/sample_n_in_oncoprint.tsv`, which is being deleted here.